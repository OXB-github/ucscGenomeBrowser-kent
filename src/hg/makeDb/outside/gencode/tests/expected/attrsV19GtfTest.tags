ENST00000430501.2	basic
ENST00000455376.2	basic
ENST00000456607.2	mRNA_end_NF
ENST00000460420.1	basic
