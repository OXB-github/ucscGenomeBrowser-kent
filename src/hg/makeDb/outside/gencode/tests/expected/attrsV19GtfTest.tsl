ENST00000430501.2	None
ENST00000455376.2	None
ENST00000456607.2	None
ENST00000460420.1	None
ENST00000464054.1	None
