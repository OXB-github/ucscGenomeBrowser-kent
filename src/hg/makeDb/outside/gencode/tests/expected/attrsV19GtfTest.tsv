geneId	geneName	geneType	transcriptId	transcriptName	transcriptType	ccdsId	level	proteinId	transcriptClass
ENSG00000236196.4	NFKBIL1	protein_coding	ENST00000430501.2	NFKBIL1-002	protein_coding		2		coding
ENSG00000236196.4	NFKBIL1	protein_coding	ENST00000455376.2	NFKBIL1-001	protein_coding		2		coding
ENSG00000236196.4	NFKBIL1	protein_coding	ENST00000456607.2	NFKBIL1-003	protein_coding		2		coding
ENSG00000236196.4	NFKBIL1	protein_coding	ENST00000460420.1	NFKBIL1-004	processed_transcript		2		nonCoding
ENSG00000236196.4	NFKBIL1	protein_coding	ENST00000464054.1	NFKBIL1-005	retained_intron		2		other
